plugins/branch_prefix/README.md

```diff
@@ -1,6 +1,6 @@
 ## Branch prefix plugin
 
-This plugin enables to add prefix to the migrated branches
+This plugin allows to add prefix to the migrated branches
```

Comment:
"This plugin adds a prefix to the names of the converted branches." to make it into more idiomatic English.